examples/misc/histogram.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
```

Comment:
Regarding to the **misc** category / path ... Should we add this category and example to `examples.json` so it appears in the `http://127.0.0.1:8080/examples/` root?

Reply:
Yes! But not yet. These examples are now for internal use, we have opened a new task to review legend examples and publish them here https://github.com/CartoDB/carto-vl/issues/948